### PR TITLE
feat: Add bullet list support to resource summary

### PR DESCRIPTION
**Pull request by katjam**
Trello card: https://trello.com/c/UkksHZe8/485-bullet-point-styling

## Description

- Add support for bullet lists in Resource summary

@Nikomus

### CODE REVIEW

**Comment by katjam** [comment]
image

**Comment by katjam** [comment]
@Nikomus Stuart has approved the copy / design changes - so please have a look to see it there is anything code-wise that could be improved with a trivial amount of work.

e.g. I'm not really happy with the faux mark-up but we already use `[ ]` to designate bold and I wanted the start and end strings to be different for extra safety - otherwise would have used `|` to enclose list.

File: src/Theme.elm [hunk 1]
```diff
@@ -413,6 +413,71 @@ generateId input =
...
+
+
+-- Helpers to tag each fragment as list of items or plain text (False)
+
+
+splitOnStartList : String -> List ( String, Bool )
+splitOnStartList richText =
+    let
+        -- First we break the rich text string into a list,
+        -- separating on <ul>, the start of a bold word or phrase
```
Comment (Nikomus) [inline src/Theme.elm]: The start of a list? Other comment below is also about bold text.

File: src/Copy/Text.elm [hunk 2]
```diff
@@ -626,7 +628,7 @@ t key =
             "The situation became dangerous and I left the house with two bags of our clothes. I rent and will probably never buy, but we are safe."
 
         HelpSelfHousingResource1Summary ->
-            "If you are worried about your safety and your economic situation, finding somewhere to live, away from the abuser, may be one of your first priorities. This information lays out some of the options that you may have and the support available to help ensure you have a safe place to live."
+            "If you are worried about your safety and your economic situation, finding somewhere to live, away from the abuser, may be one of your first priorities. This information explains: <ul>some of the options that you may have, the support available to help ensure you have a safe place to live.</ul>"
```
Comment (Nikomus) [inline src/Copy/Text.elm]: The lists have `.` at the end of the last item 🕵️

**Comment by Nikomus** [comment]
> e.g. I'm not really happy with the faux mark-up but we already use `[ ]` to designate bold and I wanted the start and end strings to be different for extra safety - otherwise would have used `|` to enclose list.

It does somewhat preclude putting a `,` in the copy if the list is in the text. If later you need commas it might be easier to just have a slot in the template for a list, and allow return of String | List String from the text system

**Review by Nikomus — DISMISSED** [review]
This does indeed put lists in the resources - looks good! 🚀

**Comment by katjam** [comment]
@Nikomus Pushed some changes in response to your comments - will await @stuartleech approval on the grammar stuff.

**Review by Nikomus — DISMISSED** [review]
OK! ⚜️

**Review by Nikomus — APPROVED** [review]
- Great! 🟪

**Pull request merged**